Classes/Hooks/TranslateHook.php

```diff
@@ -79,6 +76,10 @@ public function processTranslateTo_copyAction(string &$content, array $languageR
         $translatedContent = '';
         $targetLanguageRecord = [];
 
+        $translateConfiguration = GeneralUtility::makeInstance(TranslateOptions::class);
+        $richtextAllowTagsResolver = GeneralUtility::makeInstance(RichtextAllowTagsResolver::class);
```

Comment:
The `RichtextAllowTagsResolver` should be injected as DI/constructor argument. 

`TranslateOption` resides in `Classes/Domain/Dto` and is already excluded from DI. So `GU::mI()` is okay here for it.